### PR TITLE
Automatic libcontroller sync

### PR DESCRIPTION
**Pull request by ad-daniel**
**Description**

Proof of concept for automatic creation of PRs when a sync of libcontroller is necessary.
It's also possible to have a workflow (in libcontroller repo) that triggers when something gets merged such that it dispatches an event to this repo, triggering a workflow here which opens the PR. For our purpose however that would be overkill as the libcontroller repo gets updated at most once per day (when the nightlies are created), if even that. It's sufficient to check daily.

### CODE REVIEW

**Comment by ad-daniel** [comment]
The process seems to have worked when manually triggering it (https://github.com/cyberbotics/webots_ros2/pull/488), will check if the schedule run works too when merged.

File: .github/workflows/libcontroller_updater.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+on:
+  schedule:
+    - cron:  '0 10 * * *'
+
+jobs:
+  automatic-submodule-sync:
+    strategy:
+      matrix:
+        include:
+          - submodule_target_branch: R2022b
```
Comment (ygoumaz) [inline .github/workflows/libcontroller_updater.yml:12]: Shouldn't the target branch for master be R2022b-revision-1 on the libcontroller side ?

**Review by ygoumaz — CHANGES_REQUESTED** [review]
Very nice ! This seems to work perfectly. 

**Side note**: We may also create tags on the libcontroller repo for versions instead of branches. This way there would be a master and a develop branch there too, and versions could be accessed using tags. But we will perform this task when we change the binaries to source files instead.

File: .github/workflows/libcontroller_updater.yml (hunk 1 above)
Comment (ad-daniel) [reply .github/workflows/libcontroller_updater.yml]: yes indeed

**Review by ygoumaz — APPROVED** [review]
Perfect, thank you!

**Pull request merged**